src/00b3_survey_mics/script.R

```diff
@@ -1,7 +1,4 @@
 
-# source required functions
-source("source.R")
```

Comment:
I removed this because this is done by orderly via specifying this as a source in the `orderly.yml`